src/drivers/gps/gps.cpp

```diff
@@ -721,21 +714,6 @@ GPS::task_main()
 				memset(&_report_gps_pos, 0, sizeof(_report_gps_pos));
 
 				if (_mode == GPS_DRIVER_MODE_UBX) {
-					/* Publish initial report that we have access to a GPS,
```

Comment:
preflight check tests for the timestamp: https://github.com/PX4/Firmware/blob/master/src/modules/commander/PreflightCheck.cpp#L369
If it gets a message with timestamp=0, the check fails, as if it did not receive any message at all. So the overall behavior is unchanged, except if the check is executed <1s after boot time and the gps module already published the 0-timestamp message.